styles.css

```diff
@@ -205,3 +217,87 @@ form {
   color: #42526e;
   padding-top: 10%;
 }
+
+@media (min-width: 922px) {
```

Comment:
- [ ] Good job adding this media query breakpoint. I will suggest you use `992px` as min-width because enable your app responsiveness for other larger types of devices. Your current breakpoint is small and the responsiveness might not work well on devices lesser than `992px`

Reply:
A little mistake in the numbers, going to solve this right away!